test/e2e/generate/generate_test.go

```diff
@@ -48,136 +48,6 @@ var (
 	npPolNS = ""
 )
 
-func Test_ClusterRole_ClusterRoleBinding_Sets(t *testing.T) {
```

Comment:
Why was this test removed? I see it mentioned in the PR description but don't quite understand why.

Reply:
Kyverno default permissions should not allow setting a cluster-wide role.

I just found that there was a separate role installed only for e2e test:

https://github.com/kyverno/kyverno/blob/main/config/github/rbac.yaml

Perhaps it was to provide additional permissions for the cluster-wide roles. We can add the extra permissions there and add back this test.

Reply:
added back the test, and updated the https://github.com/kyverno/kyverno/blob/main/config/github/rbac.yaml role.